CZ/pyranda_rayleigh_taylor/rayleigh_taylor.py

```diff
@@ -516,6 +526,35 @@ def setup_argparse():
         help="Initial condition to run.",
     )
 
+    parser.add_argument(
+        "--random-velocity-magnitude",
+        type=float,
+        default=0.01,
+        help="Multiplier on random velocity field added in by Pyranda",
```

Comment:
on this and the next arg, should we set some limits/add comments on what it's doing and suggestions for reasonable values?  i.e. they're both a multiplier on 'what'

Reply:
Yeah, we can make this more rigorous; multiplier on an RT velocity scale as defined by... blah blah blah